scripts/eval/eval.py

```diff
@@ -100,7 +100,13 @@ def evaluate_model(model_cfg: DictConfig, dist_timeout: Union[float, int],
                    icl_subset_num_batches: Optional[int]):
     print(f'Evaluating model: {model_cfg.model_name}', flush=True)
     # Build tokenizer and model
-    tokenizer = build_tokenizer(model_cfg.tokenizer)
+    tokenizer_cfg: Dict[str,
+                        Any] = om.to_container(model_cfg.tokenizer,
+                                               resolve=True)  # type: ignore
+    tokenizer_name = tokenizer_cfg['name']
+    tokenizer_kwargs = tokenizer_cfg.get('kwargs', {})
+    tokenizer: AutoTokenizer = build_tokenizer(tokenizer_name,
```

Comment:
The type here should be `PreTrainedTokenizerBase`, right?

Reply:
there's an incompatability here where the `build_icl_data_and_gauntlet` uses tokenizer as an AutoTokenizer so vscode was screaming at me. I just removed the types for simplicity? Please let me know if there's a better way to resolve this.

Reply:
ah, the type in `build_icl_data_and_gauntlet` is probably just wrong. can you try fixing that type to be `PreTrainedTokenizerBase`?

Reply:
Fixed types